deploy-board/deploy_board/static/js/components/capacitycomponents.js

```diff
@@ -215,6 +215,12 @@ function getCapacityDoubleAlertMessage(isFixed) {
     return errorMessage + instruction + context;
 }
 
+function getTerminationLimitAlertMessage(isWarning) {
+    message = isWarning ? `The size you manually scaled down plus the number of hosts in terminating status exceed the specified termination limit.` 
```

Comment:
Any reason to use `message` instead of `const message`? Seems like `const message` is preferred based on the surrounding code and more limited scope